content/agents/overview.md

```diff
@@ -28,77 +28,102 @@ Cloudify includes a set of agent packages with support for:
-        # one of none, remote, init_script, provided
-        install_method: remote
-```
-
-#### Supported Installation Methods
-The following list describes the supported installation methods:
-
-* `none` - No agent is installed on the host.
-* `remote` - An agent will be installed, using SSH on Linux hosts and WinRM on Windows hosts.
-* `init_script` - An agent will be installed via a script that runs on the host when it is created. This method is only supported for specific IaaS plugins.
-* `provided` - An agent is assumed to already be installed on the host image. That agent will be configured and started via a script that runs on the host when it is created. This method is only supported for specific IaaS plugins.
-
-{{% gsNote title="Implications of Not Installing an Agent" %}}
-In some cases, you cannot or prefer not to install an agent on Cloudify-managed VMs. This might be due to a security restriction, or because a VM is a pre-configured closed appliance that you cannot access or modify. 
-In such cases, use `install_method: none` to configure Cloudify not to install an agent on the created VM. 
-However, be aware of the following implications: 
-
-* You will not be able to use plugins that assume execution on the agent's VM, meaning plugins that are configured with `executor=host_agent`. This includes Docker, Chef and Puppet plugins, among others. To work around this you must run bash or Python scripts using the [Fabric plugin]({{< relref "plugins/fabric.md" >}}) (for example, invoke the Puppet client from a script instead of using the Puppet plugin). 
-* You will not be able to install a [Diamond monitoring agent](http://diamond.readthedocs.org/) using the [Diamond plugin]({{< relref "plugins/diamond.md" >}}) because this plugin requires an agent to run. However, you can install your own monitoring agent using a cloud init / the Fabric plugin. 
-{{% /gsNote %}}
-
-
-### Pre-requisites for Linux Remote Agent Installation
+### Installation methods
 
-* SSH port (22 by default) must be open for incoming connections.
-* An SSH server must be running on the host.
+There are several methods by which the installation script is distributed
+to the agent host and executed.
 
-### Pre-requisites for Windows Remote Agent Installation
-
-* WinRM port (5985 by default) must be open for incoming connections.
-* WinRM must be enabled. Run these command in `userdata` (or something equivalent), or create an image with the following configuration:
-
-  {{< gsHighlight  bash  >}}
+* `remote` - The installation script is pushed to the agent host, and
+executed remotely using SSH on Linux hosts and WinRM on Windows hosts.
+This is the default and simplest way for systems that allow SSH/WinRM
+access. A pre-requisite for remote installation is:
+    * For Linux an SSH server must be running on the agent host, and
+    the SSH port (22 by default) must be open for incoming connections.
+    * For Windows WinRM must be enabled, and the WinRM port (5985 by
```

Comment:
"For Windows WinRM must be enabled" -> "Windows, WinRM" (comma) (same for linux)